activerecord/lib/active_record/relation.rb

```diff
@@ -3,6 +3,37 @@
 module ActiveRecord
   # = Active Record \Relation
   class Relation
+    class ExplainProxy  # :nodoc:
+      SUPPORTED_METHODS = [:count, :first, :last, :maximum, :minimum, :pluck, :sum]
```

Comment:
if we only support those methods why use method missing?

Reply:
I've updated the implementation to use regular methods. This also makes sure the method signatures match.